src/admin-views/widgets/components/input-section-start.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Admin View: Widget Input Section Start.
+ * The beginning of a container for sectioned inputs.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/input-section-start.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var string $title The (optional) section title.
+ */
+
+ $section_classes = [
+	 'tribe-events-widget-admin-form__input-section',
+ ];
```

Comment:
:warning: Found precision alignment of 1 spaces.

<sub>hash: 6b4802b21ff2fd5aecb55dddbc62e1a8</sub>